### PR TITLE
DB-2432: turn off interventions provider by default

### PR DESCRIPTION
**Pull request by aleksei-cliqz**

### CODE REVIEW

**Comment by tsl143** [comment]
Do you think it will be good idea to remove `UrlbarProviderInterventions` from `moz.build` and `jar.mn` ?

**Comment by aleksei-cliqz** [comment]
> Do you think it will be good idea to remove `UrlbarProviderInterventions` from `moz.build` and `jar.mn` ?

I do not think so for now. Because there are many other providers along the way which also have to be disabled following this case.
It is easier to control it via prefs for now.

File: mozilla-release/browser/app/profile/firefox.js [hunk 1]
```diff
@@ -371,7 +371,10 @@ pref("browser.urlbar.usepreloadedtopurls.expire_days", 14);
...
   // If true, we show actionable tips in the Urlbar when the user is searching
   // for those actions.
-  pref("browser.urlbar.update1.interventions", true);
+
+  // CLIQZ-SPECIAL: DB-2432, in Cliqz Browser this always has to be false
+  // since we use our own extension dropdown.
+  pref("browser.urlbar.update1.interventions", false);
   // If true, we show new users and those about to start an organic search a tip
   // encouraging them to use the Urlbar.
   pref("browser.urlbar.update1.searchTips", true);
```
Comment (alver-cliqz) [inline mozilla-release/browser/app/profile/firefox.js]: May be turn off this also?

Comment (alver-cliqz) [reply mozilla-release/browser/app/profile/firefox.js]: Ah, no needs. It disabled for both our builds, beta and release. Please create a ticket for next version (1.35) to check this

**Comment by alver-cliqz** [comment]
We don't need to do anything with it now, because it disabled for both our builds, beta and release. Please create a ticket for next version (1.35) to check this

**Pull request closed without merging**